FNAC.js

```diff
@@ -0,0 +1,316 @@
+{
+	"translatorID": "825f208c-eb96-4372-9d21-b879c3a910bb",
+	"label": "FNAC",
+	"creator": "César Lizurey",
+	"target": "^https?://(www\\.)?fnac\\.com",
+	"minVersion": "5.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2024-10-21 19:30:59"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2024 César Lizurey
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	// Check if the URL matches the desired format
+	var match = url.match(/^https:\/\/www\.fnac\.com\/a(\d+)\/([\w-]+)$/);
+
+	if (match) {
+		Z.debug("URL OK");
+
+		// Get the second <a> element in the breadcrumb to check the itemType
+		var secondLink = doc.querySelector('nav.f-breadcrumb > ul.f-breadcrumb__list > li:nth-of-type(2)');
+		if (secondLink) {
+			Z.debug("Breadcrumb OK");
+			switch (secondLink.textContent.trim()) {
+				case 'Livre': {
+					return 'book';
+				}
+				case 'Musique': {
+					return 'audioRecording';
+				}
+				case 'DVD & Vidéo': {
+					return 'videoRecording';
+				}
+				case 'Jeux vidéo': {
+					return 'computerProgram';
+				}
+				default: {
+					Z.debug("Unknown category '" + secondLink.textContent.trim() + "' will be ignored by Zotero");
+					Z.debug("Items in this category will be ignored by Zotero: " + secondLink.textContent.trim());
+					return false;
+				}
+			}
+		}
+
+		Z.debug("Breadcrumb KO");
+		return false;
+	}
+	
+	Z.debug("URL KO");
+	return false;
+}
+
+function doWeb(doc, url) {
+	Z.debug("Scraping from Page");
+	var item = new Zotero.Item(detectWeb(doc, url) || "book");
+	item.URL = url;
+	item.title = doc.querySelector('h1.f-productHeader-Title').textContent.trim();
```

Comment:
> I'm not seeing this on the page. And use `text()` rather than `querySelector(...).textContent.trim()`.

Indeed, the last class has changed. I modified the selector to rely on something more consistent.
I'm not familiar with the "text" method. Do you suggest this?

`item.title = doc.text('h1.f-productHeader-Title');`